Sentinel/Classes/EmailSender/MailData.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  MailData.swift
+//  Sentinel
+//
+//  Created by Antonijo Bezmalinovic on 22.11.2023..
+//
+
+import Foundation
+
+public struct Attachment {
+    
+    let data: Data?
```

Comment:
Can there be an attachment if `Data` is `nil`?
I'm unsure if that will resolve some scenarios, but if not, I would set this to `Data`.